charts/newrelic-infrastructure/values.yaml

```diff
@@ -583,4 +586,4 @@ fedramp:
 
 # -- (bool) Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog`
 # @default -- `false`
-verboseLog:
+verboseLog:
```

Comment:
Seems like the POSIX-compliant newline got removed somehow. Could we restore it?

Reply:
Done